tools/reviewer/local_server/service/code_review.proto

```diff
@@ -53,6 +53,17 @@ message PongResponse { string message = 1; }
 
 message RemoveWorkspaceRequest { string workspace_name = 1; }
 
+message GithubRepo {
+  string owner = 1;
+  string repo = 2;
+}
+
+message GithubPr {
+  GithubRepo repo = 1;
```

Comment:
Wondering - why not put `owner` and `repo` directly under `GithubPr`, and not have `GithubRepo`?

Reply:
Yes, it will be better.
Done.